gprofiler/profilers/python.py

```diff
@@ -151,6 +153,49 @@ def __init__(
         self.output_path = Path(self._storage_dir) / f"pyperf.{random_prefix()}.col"
         self.user_stacks_pages = user_stacks_pages
 
+    @staticmethod
+    def _fix_kernel_headers_symlink():
```

Comment:
This function should be documented (I suppose, with the comment in the PR's description)

Reply:
Added in the latest commit